test/models/artefact_test.rb

```diff
@@ -251,13 +248,12 @@ class ArtefactTest < ActiveSupport::TestCase
     artefact.redirect_url = "/foobar#chapter"
     assert artefact.valid?
 
-    artefact.redirect_url = "http://foo.bar/"
-    refute artefact.valid?
+    artefact.redirect_url = "http://foo.bar.gov.uk/"
```

Comment:
Yeah, the true definition of what is valid or not is in the Publishing API [1].

1: https://github.com/alphagov/publishing-api/blob/master/app/validators/routes_and_redirects_validator.rb#L165